buildstockbatch/gcp/gcp.py

```diff
@@ -843,6 +925,12 @@ def main():
         gcs_prefix = os.environ["GCS_PREFIX"]
         job_name = os.environ["JOB_NAME"]
         GcpBatch.run_task(task_index, job_name, gcs_bucket, gcs_prefix)
+    elif "POSTPROCESS" == os.environ.get("JOB_TYPE", ""):
+        gcs_bucket = os.environ["GCS_BUCKET"]
+        gcs_prefix = os.environ["GCS_PREFIX"]
+        results_dir = os.environ["RESULTS_DIR"]
+        do_timeseries = os.environ.get("DO_TIMESERIES", "False") == "True"
+        GcpBatch.run_combine_results_on_cloud(gcs_bucket, gcs_prefix, results_dir, do_timeseries)
```

Comment:
What do you think about moving these subtasks into separate scripts instead of differentiating via these environment variables? Now that three different things happening in the same `main` function, that might be more readable. (I'm okay with doing that in a separate refactoring PR though.)

Reply:
Overall, in favor, but definitely a separate PR. I'd kinda like for us to give a bit more thought to design, like using inversion of control.